README.md

```diff
@@ -0,0 +1,20 @@
+Current functions:
+StartMetrics() - starts the metrics server over an endpoint and port. Should be configurable with a sane default
```

Comment:
`starts` should be capitalized.

Reply:
`Should be configurable with a sane default` please use complete sentences in a `README`.

Reply:
Honestly, I would prefer the README not to mention all functions present in the library. Perhaps provide the easiest path and docs to showcase other methods.

Reply:
See above